oci/spec_opts.go

```diff
@@ -1264,3 +1273,32 @@ func WithDevShmSize(kb int64) SpecOpts {
 		return ErrNoShmMount
 	}
 }
+
+// freebsdLinuxEmulationMounts modifies the mount spec to mount emulated Linux filesystems on FreeBSD,
+// as per: https://wiki.freebsd.org/LinuxJails
+func freebsdLinuxEmulationMounts(s *Spec) error {
+	var mounts []specs.Mount
+	for _, m := range s.Mounts {
+		path := filepath.Clean(m.Destination)
+		if path == "/proc" {
+			mounts = append(mounts, specs.Mount{
+				Destination: "/proc",
+				Type:        "linprocfs",
+				Source:      "linprocfs",
+				Options:     []string{},
+			})
+			continue
+		}
+		if path == "/dev/fd" {
+			m.Options = append(m.Options, "linrdlink")
+		}
```

Comment:
Similar question as above; if there is no existing `fdescfs` mount at `/dev/fd`, wouldn't we want to add one and include this option?